<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/*默认字体*/
*{
	font: 18pt;
	font-weight:bold;
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWebView" name="webView" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1280</width>
      <height>800</height>
     </rect>
    </property>
    <property name="url" stdset="0">
     <url>
      <string/>
     </url>
    </property>
   </widget>
   <widget class="QPushButton" name="zoomout">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>10</y>
      <width>75</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>缩小</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fit">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>10</y>
      <width>75</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Fit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="zoomin">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>10</y>
      <width>75</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>放大</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>10</y>
      <width>75</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>添加</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="latitude">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>10</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>108</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="longitude">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>10</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>34</string>
    </property>
   </widget>
   <widget class="QPushButton" name="delete_2">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>10</y>
      <width>75</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>删除</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="name">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>10</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Local Video</string>
    </property>
   </widget>
   <widget class="QPushButton" name="delete_all">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>10</y>
      <width>75</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>删除所有</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="number">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>10</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>18792430347</string>
    </property>
   </widget>
   <widget class="QLabel" name="draw">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1280</width>
      <height>800</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="widgetCall" native="true">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>280</y>
      <width>180</width>
      <height>60</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="cancel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="call">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>拨打电话</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QCheckBox" name="check_type">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>卫星地图</string>
    </property>
   </widget>
   <zorder>draw</zorder>
   <zorder>webView</zorder>
   <zorder>zoomout</zorder>
   <zorder>fit</zorder>
   <zorder>zoomin</zorder>
   <zorder>add</zorder>
   <zorder>latitude</zorder>
   <zorder>longitude</zorder>
   <zorder>delete_2</zorder>
   <zorder>name</zorder>
   <zorder>delete_all</zorder>
   <zorder>number</zorder>
   <zorder>widgetCall</zorder>
   <zorder>check_type</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKitWidgets/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
